pkgs/development/python-modules/stack-data/default.nix

```diff
@@ -0,0 +1,57 @@
+{ asttokens
+, buildPythonPackage
+, executing
+, fetchgit
+, git
+, lib
+, littleutils
+, pure-eval
+, pygments
+, pytest
+, setuptools_scm
+, toml
+, typeguard
+}:
+
+buildPythonPackage rec {
+  pname = "stack_data";
+  version = "0.0.7";
+
+  # Install requires .git and fetchFromGitHub doesn't support leaving it so
+  # let's use fetchgit
+  src = fetchgit {
+    url = "https://github.com/alexmojaki/${pname}.git";
+    rev = "v${version}";
+    sha256 = "0smifhq8ma8qqhsawmgbfxq5i07rszgysf23i38dlal4h9krm03z";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    git
+  ];
+
+  buildInputs = [
+    setuptools_scm
+    toml
+  ];
```

Comment:
```suggestion
  nativeBuildInputs = [
    git
    setuptools_scm
    toml
  ];
```

Reply:
I'm always very confused if packages should go to `buildInputs` or `nativeBuildInputs`. The confusion is even greater when it seems that the package builds just fine either way.. But I now made this fix. Thanks!